orion-server/src/main/java/com/pinterest/orion/server/api/AMI.java

```diff
@@ -0,0 +1,25 @@
+package com.pinterest.orion.server.api;
+
+public class AMI {
+  private final String amiId;
+  private final String applicationEnvironment;
+  private final String creationDate;
+
+  public AMI(String amiId, String applicationEnvironment, String creationDate) {
+    this.amiId = amiId;
+    this.applicationEnvironment = applicationEnvironment;
+    this.creationDate = creationDate;
+  }
```

Comment:
I suggest you add basic formatting checks here. I assume amiId needs to follow some standards. 

CreationDate is a timestamp. The timestamp can have different formats.

Reply:
This class is used to populate the list of images returned by AWS and send to the UI. So it doesn't make sense to check format here.

Reply:
This timestamp is UTC, btw.

Reply:
Can we make it clear in the code that timestamp is UTC? using Javadoc perhaps?

Reply:
Done